### ISSUE TITLE
Make message list easier to look at

### ISSUE THREAD

**Issue by thstyl2000** [nextcloud/mail#10393]
### Is your feature request related to a problem? Please describe.

Currently it feels to me that the message list is a bit too cluttered.

### Describe the solution you'd like

Allow extra space between each message, increase subject font size. Here is my futile attempt to make it more to my liking. I think it is definitely better, but a few details still need to be adjusted (like the size of avatars and so on):

```
.envelope__preview-text {
    width:97%;
    font-size:80%;
}

.envelope__subtitle {
    font-size:118%;
}

.list-item__wrapper {
    margin-top:0.5em;
    margin-bottom:0.5em;
}

.list-item-content {
    margin-left:0.5em;
}


.list-item-details__details {
    padding-bottom:0.1em;
}

.list-item-details__extra {
    padding-bottom:0.3em;
}
```

**Comment by ChristophWurst** [comment]
Thank you for the suggestions @thstyl2000!

@nimishavijay what do you think?

**Comment by nimishavijay** [comment]
Nice! Could you share a before/after screenshot for easier reviewing? 

I would also recommend making use of variables like `calc(var(--default-grid-baseline)*2)` instead of hardcoded values like `0.5em` to be consistent :)

**Comment by thstyl2000** [comment]
If you can give me a hint about creating ~20 test emails on a dovecot imap server, then yes, absolutely! Otherwise I would need to strike out all sensitive data from my own inbox and it wouldn't really help (you'd just see a bunch of red lines.....)

**Comment by mat-m** [comment]
> If you can give me a hint about creating ~20 test emails on a dovecot imap server, then yes, absolutely! Otherwise I would need to strike out all sensitive data from my own inbox and it wouldn't really help (you'd just see a bunch of red lines.....)

Create a temporary folder in one of your accounts. Send you 20 test messages and you're done :). I personnaly use newsletters from shops, they are not that sensitive :) 

About spacing in the messages list: It"s a matter of preferences. Some people want to see the maximum items (compact view) and other want readable items (clear view).
You can override current settings with Custom Stylesheet plugin; that's what I do to compact the account & folders list.

**Comment by thstyl2000** [comment]
Got a couple screenshots of my spam folder :)

Before (standard):

![Image](https://github.com/user-attachments/assets/89ba3cb7-c9c5-4a62-a600-4cb00022df09)



After:

![Image](https://github.com/user-attachments/assets/853391e5-6de9-4361-9d47-1f6ea5f58bf2)




The effect is subtle, so it's not like having a compact or clear view, more like having the important infos stick out. For me it is just way less tiring if I work with emails a lot.

 For a more compact view I think one should choose to shrink all fonts uniformly.

**Comment by mat-m** [comment]
SO basically, you shrunk body font size and increased subject font size, right ?
Quite efficient for title, yes !

**Comment by thstyl2000** [comment]
Yes, and added a little bit of space around the text box so that it looks a bit less "crowded" and there is more space between messages.

**Comment by nimishavijay** [comment]
Thank you for the screenshots @thstyl2000 !

Indeed the font brings more attention to the subject. However, we use a pretty standard set of font sizes with little variability except for headings, so I would stay away from altering those to make sure things remain consistent :)

**Comment by thstyl2000** [comment]
Yes, I see the point. What about making the message preview line a bit transparent.....?

PS just noticed that the avatar dots and the three dot buttons are not vertically centered, is that on purpose?
